setup_cfg_fmt.py

```diff
@@ -316,6 +316,44 @@ def _is_ok_classifier(s: str) -> bool:
     return [s for s in classifiers if _is_ok_classifier(s)]
 
 
+def _imp_classifiers(setup_cfg: str) -> Optional[str]:
+    tox_config_to_names = {
+        'py': 'CPython',
+        'pypy': 'PyPy',
+        'jython': 'Jython',
+    }
+
+    cfg = configparser.ConfigParser()
+    cfg.read(setup_cfg)
+    classifiers = cfg.get('metadata', 'classifiers', fallback='')
+    tags = []
+
+    for classifier in classifiers.strip().splitlines():
+        if classifier.startswith(
+            'Programming Language :: Python :: Implementation :: ',
+        ):
+            tags.append(classifier.split(' :: ')[-1])
+
+    tox_ini = _adjacent_filename(setup_cfg, 'tox.ini')
+    if os.path.exists(tox_ini):
+        cfg = configparser.ConfigParser()
+        cfg.read(tox_ini)
+
+        envlist = cfg.get('tox', 'envlist', fallback='')
+        if envlist:
+            for env in envlist.split(','):
+                for key, tag in tox_config_to_names.items():
+                    if (key in env) and (tag not in tags):
+                        tags.append(tag)
+
+    classifiers_list = [
+        f'Programming Language :: Python :: Implementation :: {name}'
+        for name in tags
+    ]
+
+    return '\n'.join(classifiers_list)
```

Comment:
I would probably `return '\n'.join(generator expression)` personally -- the intermediate list isn't used anywhere